portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/snippets/siteStatus-snippet.vm

```diff
@@ -8,7 +8,25 @@
 
     <div id="siteStatus" class="Mrphs-siteStatus Mrphs-messagePanel__information is-unpublished">
         <h4>${rloader.sit_unpublished}</h4>
-        <button onclick="publishSite('${sitePages.siteId}');" class="Mrphs-button--active">${rloader.sit_publish_now}</button>
+        <button class="Mrphs-button--active" data-toggle="modal" data-target="#publishSiteConfirmModal">${rloader.sit_publish_now}</button>
+    </div>
+
+    <div id="publishSiteConfirmModal" class="modal fade" role="dialog" tabindex="-1" aria-hidden="true">
```

Comment:
Is the `.fade` class necessary?

Reply:
I tested here without the .fade and works too:
https://www.w3schools.com/bootstrap/bootstrap_modal.asp

Should we remove it?

Reply:
Removing it would be my preference, please.